main.md

```diff
@@ -564,6 +564,17 @@ JWS `alg` values the authorization server supports for DPoP proof JWTs.
 :   A JSON array containing a list of the JWS `alg` values supported
 by the authorization server for DPoP proof JWTs. 
 
+## Client Registration Metadata {#client-meta}
+
+This document introduces the following new client registration metadata
+[@RFC7591] parameter to indicate that the client always uses
+DPoP when requesting tokens from the authorization server.
+
+`always_uses_dpop`
+:   Boolean value specifying whether the client always uses DPoP for token requests.  If omitted, the default value is `false`.
+
+If `true`, the authorization server MUST reject token requests from this client that do not contain the DPoP header.
+
```

Comment:
```suggestion
## Client Metadata {#client-meta}

The Dynamic Client Registration Protocol [@RFC7591] defines an API for dynamically registering OAuth 2.0 client metadata with authorization servers. The metadata defined by [@RFC7591], and registered extensions to it, also imply a general data model for clients that is useful for authorization server implementations even when the Dynamic Client Registration Protocol isn't in play. Such implementations will typically have some sort of user interface available for managing client configuration. The following client metadata parameter is introduced by this document to indicate that the client always uses DPoP when requesting tokens from the authorization server.

`always_uses_dpop`
:   Boolean value specifying whether the client always uses DPoP for token requests.  If omitted, the default value is `false`.

If `true`, the authorization server MUST reject token requests from this client that do not contain the DPoP header.
```

Reply:
The suggestion above is meant to replace the whole subsection. The change is based on a and meant to address [this piecemeal conversation](https://github.com/danielfett/draft-dpop/pull/109#issuecomment-1043406960) with @danielfett

Reply:
The third commit https://github.com/danielfett/draft-dpop/pull/109/commits/63fab689a42fc79b77ebdf98a73b41f7b4d69ab7, which I apparently made in parallel with your suggestion above, does exactly what you requested, modulo different line breaks.

Are we good to merge?

Reply:
Damn, that was confusing... but yeah, I think we are we good to merge this one.

Reply:
Can I get an approval?

Reply:
Yep - I understand we may tweak the meaning but I think doing that with `always_use_dpop` and the PAR text in the same draft will be easier.